cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -44,7 +49,10 @@ class Node0Impl extends TIRDataFlowNode0 {
   Instruction asInstruction() { result = this.(InstructionNode0).getInstruction() }
 
   /** Gets the operands corresponding to this node, if any. */
-  Operand asOperand() { result = this.(OperandNode0).getOperand() }
+  Operand asOperand() {
+    result = this.(MultipleUseOperandNode0).getOperand() or
+    result = this.(SingleUseOperandNode0).getOperand()
```

Comment:
Would this be equivalent?
```suggestion
    result = this.(OperandNode0).getOperand()
```

Reply:
Good point. That should definitely work.